Thanks to Fernando Lopez-Lezcano <nando@ccrma.stanford.edu>
Reviewed by speps <speps at aur dot archlinux dot org>

diff -aur dssi-vst-0.9.2/Makefile dssi-vst-0.9.2.patched/Makefile
--- dssi-vst-0.9.2/Makefile	2010-05-04 11:01:29.000000000 +0200
+++ dssi-vst-0.9.2.patched/Makefile	2014-03-28 18:24:41.128287015 +0100
@@ -38,12 +38,15 @@
 all:		$(TARGETS)
 
 install:	all
+		mkdir -p $(DSSIDIR)
 		mkdir -p $(DSSIDIR)/dssi-vst
 		mkdir -p $(LADSPADIR)
 		mkdir -p $(BINDIR)
 		install dssi-vst.so $(DSSIDIR)
 		install dssi-vst.so $(LADSPADIR)
-		install dssi-vst-server.exe.so dssi-vst-server dssi-vst-scanner.exe.so dssi-vst-scanner dssi-vst_gui $(DSSIDIR)/dssi-vst
+		install dssi-vst-server.exe.so dssi-vst-scanner.exe.so dssi-vst_gui $(DSSIDIR)/dssi-vst
+		install dssi-vst-server.exe $(DSSIDIR)/dssi-vst/dssi-vst-server
+		install dssi-vst-scanner.exe $(DSSIDIR)/dssi-vst/dssi-vst-scanner
 		install vsthost $(BINDIR)
 
 clean:
